bot/src/BlockSync.ts

```diff
@@ -132,7 +132,7 @@ export class BlockSync {
 
     this.accountMiners = new AccountMiners(
       this.accountset,
-      startingMinerState.filter(x => x.cohortId !== undefined) as any,
+      startingMinerState.filter(x => x.seat !== undefined) as any,
```

Comment:
How did cohortId get renamed to seat? That's a new concept.

Reply:
It's just nested under a seat object